### ISSUE TITLE
Please add an option in the menubar for removing duplicate lines

### ISSUE THREAD

**Issue by desbest** [ilyakam/RemoveDuplicateLines#21]
Hello
Please add an option in the menubar for removing duplicate lines
It would be nice if we could click an option in the menu to do this.

**Comment by ilyakam** [comment]
Hey @desbest!

Thanks for the feature request. I'm going to work on that shortly.

In the meantime, here's a workaround to add it to the Edit context menu, below the "Sort Lines" commands:

<img width="467" alt="demo" src="https://user-images.githubusercontent.com/183227/100297710-216ceb80-2f44-11eb-9812-3e6f4ed6fb8f.png">

1. Open Sublime Text 3
1. Go to Preferences ▸ Browse Packages…
1. Open the "User" folder
1. Create a new `Main.sublime-menu` file (or edit an existing one) with the following content:
    ```json5
    [
      {
        "id": "edit",
        "children": [
          {
            "caption": "Remove Duplicate Lines",
            "command": "remove_duplicate_lines",
            "id": "sort_lines",
            "mnemonic": "r"
          }
        ]
      }
    ]
    ```

**Comment by ilyakam** [comment]
Released in [v2.1.0](https://github.com/ilyakam/RemoveDuplicateLines/releases/tag/2.1.0), cheers!
